README.md

```diff
@@ -10,6 +10,10 @@ Example of an autonomous service using the [Open Autonomy](https://docs.autonola
 - [Docker Engine](https://docs.docker.com/engine/install/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - The [Open Autonomy](https://docs.autonolas.network/open-autonomy/guides/set_up/#set-up-the-framework) framework
+- The [Open Autonomy Cryptography Plugin](https://pypi.org/project/open-aea-ledger-ethereum) for ethereum.
+    
+          pip install open-aea-ledger-ethereum
+
```

Comment:
This is already in the `Pipenv` and should be taken care of in [this](https://github.com/valory-xyz/hello-world/blob/v0.1.3/README.md?plain=1#L22) step.

Reply:
No. The README.md is for developing based on this repository. The `docs` is what is published in the documentation website, and it is a lightweight version of the README. 

The process there (as for the rest of the kits) is operator-oriented: is about fetching a hash and deploying from the downloaded service.yaml using the framework CLI, they do not need to clone the repository.

Therefore, they don't have the Pipenv available.

Reply:
Included in this PR https://github.com/valory-xyz/open-autonomy/pull/2175

Therefore, we can close this PR.